packages/bpk-stylesheets/fonts.scss

```diff
@@ -0,0 +1,104 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@mixin bpk-define-font-stack($baseURL, $font-family-stack) {
+  :global(body) {
+    font-family: $font-family-stack;
```

Comment:
Why not just leavel the existing definition that defined this in place instead of doing it via the mixin? In fact do we even need the mixin anymore?